frontend/src/components/AddEvent.tsx

```diff
@@ -1,16 +1,88 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import {EventModel} from "../model/EventModel";
+import {User} from "../model/User";
+import './AddEvent.css';
+import EventForm from "./EventForm";
 
 type Props = {
-    events:EventModel[]
+    user: User | undefined,
+    events: EventModel[],
+    getEventsByCreator: (creator: string) => void,
+    saveEvent: (event: EventModel) => void,
+    updateEvent: (id: string, event: EventModel) => void,
+    deleteEvent: (event: EventModel, callback?: () => void) => void
 }
 
-function AddEvent(props:Props) {
+function AddEvent(props: Props) {
+
+    const [isFormVisible, setIsFormVisible] = useState(false);
+    const [selectedEvent, setSelectedEvent] = useState<EventModel | undefined>(undefined);
+
+    useEffect(() => {
+        if (props.user) {
+            props.getEventsByCreator(props.user.username);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [props.user?.username]);
+
+    function onDeleteHandler(event: EventModel) {
+        console.log("Delete button clicked");
```

Comment:
Do you use these console.logs somewhere? If they are only for development then they should be deleted.